microcosm_flask/conventions/crud_adapter.py

```diff
@@ -42,6 +42,10 @@ def search(self, offset, limit, **kwargs):
         count = self.store.count(**kwargs)
         return items, count
 
+    def count(self, offset=None, limit=None, **kwargs):
```

Comment:
I added `offset` and `limit` with default values here to ensure they are never passed to the count function. This allows us to reuse search schemas for count queries.